tests/tracking/test_tracking.py

```diff
@@ -847,3 +848,76 @@ def test_search_runs_multiple_experiments():
     assert len(MlflowClient().search_runs(experiment_ids, "metrics.m_1 > 0", ViewType.ALL)) == 1
     assert len(MlflowClient().search_runs(experiment_ids, "metrics.m_2 = 2", ViewType.ALL)) == 1
     assert len(MlflowClient().search_runs(experiment_ids, "metrics.m_3 < 4", ViewType.ALL)) == 1
+
+
+@pytest.mark.skipif(
+    "MLFLOW_SKINNY" in os.environ,
+    reason="Skinny client does not support the np or pandas dependencies",
+)
+def test_log_table():
```

Comment:
Can we add a test case where a table is logged in a subdirectory?

```
mlflow.log_table(table, artifact_file="dir/foo.csv")
mlflow.log_table(table, artifact_file="dir/foo.csv")
```

Reply:
Can we also add a test case where new and existing tables have different columns?

```python
mlflow.log_table({"a": [0]}, artifact_file="foo.csv")
mlflow.log_table({"b": [1, 2, 3]}, table, artifact_file="foo.csv")
```